study-builder/studies/rarex/i18n/en.json

```
@@ -191,15 +191,19 @@
   "general_information_ethnicity_other": "Other",
   "general_information_ethnicity_other_details": "Please describe you ethnicity",
 
-  "general_information_gender_identity": "What terms best express how you describe your gender identity?",
-  "general_information_patient_gender_identity": "What terms best describe the patient’s gender identity?",
+  "general_information_gender_identity": "What term best describes your gender identity?",
+  "general_information_patient_gender_identity": "What term best describes the patient's gender identity?",
   "general_information_gender_identity_validation_hint": "Gender cannot be empty",
   "general_information_gender_identity_man": "Man",
   "general_information_gender_identity_woman": "Woman",
+  "general_information_gender_identity_boy": "Boy",
+  "general_information_gender_identity_girl": "Girl",
   "general_information_gender_identity_nonbinary": "Nonbinary",
   "general_information_gender_identity_transgender": "Transgender",
   "general_information_gender_identity_none_of_above": "None of these describe me, and I’d like to describe myself",
   "general_information_gender_prefer_not_to_answer": "Prefer not to answer",
+  "general_information_gender_other_detail": "Other gender specify",
+  "general_information_gender_other_detail_req_hint": "Please specify other gender",
```

I think we should add `*` to indicate that it's required.
```suggestion
  "general_information_gender_other_detail_req_hint": "Please specify other gender *",
```

---

It's a validation message. I'll add * to the placeholder